front/src/modules/settings/data-model/objects/SettingsObjectDisabledMenuDropDown.tsx

```diff
@@ -44,7 +57,7 @@ export const SettingsObjectDisabledMenuDropDown = ({
           </DropdownMenu>
         }
         dropdownHotkeyScope={{
-          scope: scopeKey + '-settings-object-disabled-menu-dropdown',
+          scope: dropdownScopeId,
```

Comment:
I am unsure it's a good practice to mix scopeIds and hotkeyScopeIds, it looks to be fine from a behavior perspective but I fear developers will confuse both